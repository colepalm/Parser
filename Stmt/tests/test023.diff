1c1
< Valid syntax
---
> Invalid syntax at (11,1), )
