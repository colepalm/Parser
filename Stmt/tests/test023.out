Invalid syntax at (11,1), )
